tests/TestFailureNaNTest.cpp

```diff
@@ -37,7 +37,7 @@ const char* failFileName = "fail.cpp";
 
 static double zero = 0.0;
 static double one = 1.0;
-static const double not_a_number = zero / zero;
+static const double not_a_number = NAN;
 static const double infinity = one / zero;
 
 extern "C" {
```

Comment:
Changing the test like this is not acceptable because it introduce platform specific ( `\<math.h\>` ) into CppUtest code. The above change will heal this failure.